src/deepEqual.ts

```diff
@@ -41,6 +41,9 @@ const deepEqual = (a: any, b: any) => {
   if (typeof a !== typeof b)
     return false
 
+  if ((a === null || b === null) && a !== b)
```

Comment:
No big deal, but the `a !== b` is redundant, I believe (since it's already established by the first if).

Reply:
Yeah did that too late in the night. I fix that before the realease.